spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/procedures/CreateChangelogViewProcedure.java

```diff
@@ -210,7 +210,12 @@ private Dataset<Row> removeCarryoverRows(Dataset<Row> df, boolean netChanges) {
     }
 
     Column[] repartitionSpec =
-        Arrays.stream(df.columns()).filter(columnsToKeep).map(df::col).toArray(Column[]::new);
+        Arrays.stream(df.columns())
+            .filter(columnsToKeep)
+            .map(CreateChangelogViewProcedure::delimitedName)
+            .map(df::col)
+            .toArray(Column[]::new);
```

Comment:
Do we need similar change here? 
https://github.com/apache/iceberg/blob/6e2eaf02f380ed8e3be7f58273d680b766897b18/spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/procedures/CreateChangelogViewProcedure.java#L182-L182